src/main/resources/application.properties

```diff
@@ -17,4 +17,4 @@ spring.jpa.properties.hibernate.event.merge.entity_copy_observer=allow
 # pagination defaults
 pagination.default.pageSize=20
 pagination.default.pageNumber=0
-
+logging.level.web=debug
```

Comment:
👍

Reply:
dunno if to remove or not, it is super handy when developing....

Reply:
configuring logging levels is standard in the sector. Leave it be. I was pleasantly surprised you thought of it.